linode/manager
# Fix Apache Airflow File Name

## Description 📝

- Renames `packages/manager/public/assets/white/apacheAirflow.svg` to `packages/manager/public/assets/white/apacheairflow.svg` to prevent 404s in some environments 
- Fix for https://github.com/linode/manager/pull/8734

## How to test 🧪

- View the internal preview link of this PR  and verify that viewing the App Details Drawer for **Apache Airflow** does **not** show a broken image
